lms/djangoapps/branding/views.py

```diff
@@ -102,3 +114,175 @@ def courses(request):
+    c) Open Sans web fonts
+
+    Example: Retrieving the footer as JSON
+
+        GET /api/branding/v1/footer
+        Accepts: application/json
+
+        {
+            "navigation_links": [
+                {
+                  "url": "http://example.com/about",
+                  "name": "about",
+                  "title": "About"
+                },
+                # ...
+            ],
+            "social_links": [
+                {
+                    "url": "http://example.com/social",
+                    "name": "facebook",
+                    "icon-class": "fa-facebook-square",
+                    "title": "Facebook"
+                },
+                # ...
+            ],
+            "mobile_links": [
+                {
+                    "url": "http://example.com/android",
+                    "name": "google",
+                    "image": "http://example.com/google.png",
+                    "title": "Google"
+                },
+                # ...
+            ],
+            "legal_links": [
+                {
+                    "url": "http://example.com/terms-of-service.html",
+                    "name": "terms_of_service",
+                    "title': "Terms of Service"
+                },
+                # ...
+            ],
+            "openedx_link": {
+                "url": "http://open.edx.org",
+                "title": "Powered by Open edX",
+                "image": "http://example.com/openedx.png"
+            },
+            "logo_image": "http://example.com/static/images/default-theme/logo.png",
+            "copyright": "EdX, Open edX, and the edX and Open edX logos are \
+                registered trademarks or trademarks of edX Inc."
+        }
+
+
+    Example: Retrieving the footer as HTML
+
+        GET /api/branding/v1/footer
+        Accepts: text/html
+
+
+    Example: Including the footer with the "Powered by OpenEdX" logo
+
+        GET /api/branding/v1/footer?show-openedx-logo=1
+        Accepts: text/html
+
+
+    Example: Retrieving the footer in a particular language
+
+        GET /api/branding/v1/footer?language=en
+        Accepts: text/html
+
+    Example: Retrieving the footer with all JS and CSS dependencies (for testing)
+
+        GET /api/branding/v1/footer?include-dependencies=1
+        Accepts: text/html
+
+    """
+    if not branding_api.is_enabled():
+        raise Http404
+
+    # Use the content type to decide what representation to serve
+    accepts = request.META.get('HTTP_ACCEPT', '*/*')
+
+    # Show the OpenEdX logo in the footer
+    show_openedx_logo = bool(request.GET.get('show-openedx-logo', False))
+
+    # Include JS and CSS dependencies
+    # This is useful for testing the end-point directly.
+    include_dependencies = bool(request.GET.get('include-dependencies', False))
+
+    # Override the language if necessary
+    language = request.GET.get('language', translation.get_language())
+
+    with translation.override(language):
```

Comment:
optional: move this call to just above the calls on L270 / L283 and unindent everything a level.  I think this would make the code read a bit clearer, even if it means duplicating a line once.